File: benchmarks/go/sum_loop.go
package main

import "os"

func main() {
	iters := 2400000
	acc := 0
	for i := 0; i < iters; i++ {
		acc = (acc + (i & 1023)) & 1048575
	}
	if acc == -1 {
		os.Exit(1)
	}
}

File: benchmarks/rust/sum_loop.rs
fn main() {
    let iters: i32 = 2_400_000;
    let mut acc: i32 = 0;
    for i in 0..iters {
        acc = (acc + (i & 1023)) & 1_048_575;
    }
    if acc == -1 {
        std::process::exit(1);
    }
}
